q2cli/tests/test_cli.py

```diff
@@ -77,6 +77,18 @@ def test_show_importable_types(self):
         self.assertTrue('Kennel[Dog]' in result.output)
         self.assertTrue('Mapping' in result.output)
 
+    def test_show_importable_formats(self):
+        result = self.runner.invoke(
+            tools, ['import', '--show-importable-formats'])
+        self.assertEqual(result.exit_code, 0)
+        self.assertTrue('FourIntsDirectoryFormat' in result.output)
+        self.assertTrue('IntSequenceDirectoryFormat' in result.output)
+        self.assertFalse('DummyFormat' in result.output)
```

Comment:
The new format names are `UnimportableFormat` and `UnimportableDirectoryFormat`, can you assert that both of those aren't in the output?